### PR TITLE
Added Managers

### PR DESCRIPTION
**Pull request by RithChubbyChupps**

### CODE REVIEW

File: app/_data/managers.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  grade: 12
+
+-
+
+  position: Finance Lead
+  fname: Nidhin
+  lname: Madhu
+  grade: 11
+
+-
```
Comment (akhil99) [inline app/_data/managers.yml]: what's with the inconsistent spacing between entries?

File: app/_data/managers.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  position: Webmaster, ThinkFIRST Program Coordinator, LIDAR Team Lead 
+  fname: Rithvik
+  lname: Chuppala
+  grade: 9
```
Comment (akhil99) [inline app/_data/managers.yml]: last I checked, you're not a freshman.

File: app/_data/managers.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+-
+  position: Coding Lead, LIDAR Team Lead
```
Comment (akhil99) [inline app/_data/managers.yml]: "Software Development Lead"

File: app/_data/managers.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  grade: 12
+-
+  position: Wiring Lead
+  fname: Sankalp
+  lname: Agrawal
+  grade: 12
+
+-
+
+  position: Finance Lead
```
Comment (akhil99) [inline app/_data/managers.yml]: Has this been confirmed with Kalari?

File: app/_data/managers.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  fname: Nidhin
+  lname: Madhu
+  grade: 11
+
+-
+  position: Finance Lead
```
Comment (akhil99) [inline app/_data/managers.yml]: Has this been confirmed with Kalari?

File: app/about/people.html [hunk 6]
```diff
@@ -33,6 +34,21 @@
   </ul>
 </section>
 
+<section id="managers">
+  <h2>Our Managers</h2>
+  <ul class="people-list officers-list">
+  {% for officer in site.data.managers %}
```
Comment (akhil99) [inline app/about/people.html]: rename variable please - it shouldn't be "officer"

File: app/about/people.html [hunk 7]
```diff
@@ -33,6 +34,21 @@
...
+    <li class="person officer" data-img-url="{{ officer.lname | append: ' ' | append: officer.fname | replace: ' ','-' | downcase | append: '.jpg' }}">
+      <img src="/img/people/{{officer.fname | append: '_' | append: officer.lname | append: '.jpg' }}" onerror="this.src='/img/people/generic.jpg'">
```
Comment (akhil99) [inline app/about/people.html]: are these as big as the officer photos? IMO they should be about 1/2 or 2/3 the size.

File: app/_data/managers.yml (hunk 4 above)
Comment (RithChubbyChupps) [inline app/_data/managers.yml]: confirmed by saurav

File: app/_data/managers.yml (hunk 5 above)
Comment (RithChubbyChupps) [inline app/_data/managers.yml]: confirmed with saurav

File: app/about/people.html (hunk 7 above)
Comment (RithChubbyChupps) [inline app/about/people.html]: all pics are same size

File: app/_includes/footer.html [hunk 8]
```diff
@@ -14,21 +14,24 @@
...
-          <li><a href="/about/people/">People</a></li>
+          <li><a href="/resources/links/">Helpful Links</a></li>
         </ul>
       </li>
       <li>
-        <a href="/resources/">Resources</a>
+        <h6>SOCIAL MEDIA</h6>
         <ul>
-          <li><a href="/resources/links/">Helpful Links</a></li>
+          <li><a href="https://twitter.com/mvrt115">Twitter</a></li>
```
Comment (akhil99) [inline app/_includes/footer.html]: can we add social media images as well?

File: app/about/people.html [hunk 9]
```diff
@@ -33,6 +34,21 @@
   </ul>
 </section>
 
+<section id="managers">
+  <h2>Our Managers</h2>
+  <ul class="people-list managers-list">
+  {% for officer in site.data.managers %}
```
Comment (akhil99) [inline app/about/people.html]: you still haven't changed this variable - can you do that?

**Comment by akhil99** [comment]
This PR contains commits outside its scope. Please fix and re-PR.

**Pull request closed without merging**